Things that we do

patcha-d3 opened the pull request:
Apply grid layout on Things that we do section

patcha-d3 committed:
Update hero banner section on HTML & CSS

patcha-d3 committed:
Add font link to be used in HTML

patcha-d3 committed:
apply font style .highlight and em to css

patcha-d3 committed:
Apply grid layout on Things that we do section

rosielee09 reviewed (changes requested):
Hi Pat! you added correct HTML tags for grid system! Thanks.
However, for CSS, 
'box-sizing: border-box;' is already defined at the top of the CSS file. so it would be just unnecessary repeated. Also, font-style:normal... I think most font-style is normal so you don't really need defined. Try without it and let me know if things are still working!

patcha-d3 commented:
Hi Rosie!
thank you for reviewing mine! I have tried following your suggestion, here are the results:

1. I removed _box-sizing: border-box_ from our section classes and they were still work
2. you're totally right it was unnecessary to use _font-style: normal_ when they were already normal, but turned out i still need to apply for  _em_ since its default property was italic. I might need to keep that applied :)

patcha-d3 committed:
Remove box-sizing and font-style

After feedback, I remove box-sizing and font-style as suggested. (But left em { font-style: normal } since the default value of em was italic.

patcha-d3 committed:
Removed unnecessary box-sizing and font-style

patcha-d3 committed:
Remove unnecessary box-sizing and font-style

patcha-d3 committed:
removed }
